### ISSUE TITLE
Neural Networks

### ISSUE THREAD

**Issue by ZoeMcFife** [ZoeMcFife/Zoes-Hagenberg-2-Electric-Boogaloo#73]
[Moodle](https://elearning.fh-ooe.at/mod/assign/view.php?id=719255)

(tensor flow documentation for acatiobtrion functions and optimizers)
(use drop outs and batch normilsatzion)

**Comment by ZoeMcFife** [comment]
this is such a boring and ass homework guh

**Comment by ZoeMcFife** [comment]
talk about vanishing gradient effect

**Comment by ZoeMcFife** [comment]
underfitting overfitting
